sdk/batch/azure-compute-batch/src/test/java/com/azure/compute/batch/JobScheduleTests.java

```diff
@@ -76,19 +76,19 @@ public void canCRUDJobSchedule() throws Exception {
             // UPDATE
             LinkedList<MetadataItem> metadata = new LinkedList<MetadataItem>();
             metadata.add((new MetadataItem("key1", "value1")));
-            BatchJobScheduleUpdateParameters jobScheduleUpdateParameters = new BatchJobScheduleUpdateParameters();
-            jobScheduleUpdateParameters.setMetadata(metadata);
-            jobScheduleClient.patch(jobScheduleId, jobScheduleUpdateParameters);
+            BatchJobScheduleUpdateOptions jobScheduleUpdateOptions = new BatchJobScheduleUpdateOptions();
```

Comment:
Looks like I did not explicitly test the JobSchedule PUT API (replaceJobSchedule). We should include a snippet here to explicitly test replaceJobSchedule with the converged JobSchedule object from a prior GET request and updating some updatable properties directly on that object.